### PR TITLE
ARCADSoftware.DOT-Anonymizer version Version 23.1.0-SNAPS

### PR DESCRIPTION
**Pull request by sancoil**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98769)

### CODE REVIEW

File: manifests/a/ARCADSoftware/DOT-Anonymizer/Version 23.1.0-SNAPS/ARCADSoftware.DOT-Anonymizer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ARCADSoftware.DOT-Anonymizer
+PackageVersion: Version 23.1.0-SNAPS
+PackageLocale: en-US
+Publisher: ARCAD Software
+PublisherUrl: https://www.dot-anonymizer.fr/
```
Comment (stephengillie) [inline manifests/a/ARCADSoftware/DOT-Anonymizer/Version 23.1.0-SNAPS/ARCADSoftware.DOT-Anonymizer.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.arcadsoftware.fr/
```

File: manifests/a/ARCADSoftware/DOT-Anonymizer/Version 23.1.0-SNAPS/ARCADSoftware.DOT-Anonymizer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: ARCADSoftware.DOT-Anonymizer
+PackageVersion: Version 23.1.0-SNAPS
+PackageLocale: en-US
+Publisher: ARCAD Software
+PublisherUrl: https://www.dot-anonymizer.fr/
+PublisherSupportUrl: https://www.dot-anonymizer.com/
+Author: Arcad software
+PackageName: DOT-Anonymizer
+PackageUrl: https://www.arcadsoftware.fr/
```
Comment (stephengillie) [inline manifests/a/ARCADSoftware/DOT-Anonymizer/Version 23.1.0-SNAPS/ARCADSoftware.DOT-Anonymizer.locale.en-US.yaml:12]:
```suggestion
PublisherUrl: https://www.arcadsoftware.fr/
PublisherSupportUrl: https://www.dot-anonymizer.com/
Author: Arcad software
PackageName: DOT-Anonymizer
PackageUrl: https://www.dot-anonymizer.fr/masquage-des-donnees-de-test/dot-anonymizer/
```

**Comment by stephengillie** [comment]
Hi @sancoil,

I'm not able to find this InstallerUrl from the PackageUrl. Browsing the site, the package doesn't appear readily available for download. The InstallerUrl seems to be the developer's internal Artifactory server, and since the URL no longer works, it appears they either removed the server, or removed its public reachability. Is there another page on their site that has a link to the package?

**Comment by Trenly** [comment]
Close with reason: Stale PR;

**Pull request closed without merging**